source/guide_mailman-3.rst

```diff
@@ -346,26 +351,17 @@ for Mailman 2 and is based on the script provided in the official installation i
  if [ $# = 2 ]
  then
    i=$2
-   p=61902 # Change this - first free port
+   p=8024 
+   h=isabell.local.uberspace.de
```

Comment:
```suggestion
   h=${USER}.local.uberspace.de
```

this should work fine and doesn't require any manual changes.

Reply:
Even better!